deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -136,3 +210,31 @@ def detect_whodata_start(file_monitor):
     """
     file_monitor.start(timeout=60, callback=generate_monitoring_callback(fim.CB_REALTIME_WHODATA_ENGINE_STARTED),
                        error_message=fim.ERR_MSG_WHODATA_ENGINE_EVENT)
+
+
+def detect_windows_sacl_configured(file_monitor, file):
+    """Detects when windows permision checks have been configured for a given file.
+
+    Args:
+        file_monitor (FileMonitor): file log monitor to detect events
+        file: The path of the file that will be monitored
+    """
+
+    pattern = fr".*win_whodata.*The SACL of '({file})' will be configured"
+
+    file_monitor.start(timeout=60, callback=generate_monitoring_callback(pattern),

```

Comment:
Use non hardcoded timeout

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3627/commits/190d24a2bbe08cc0d27afce12936ad8270bb6567